ShanghaiBloodSports/Assets/Scripts/Moves/MoveTrinket.cs

```diff
@@ -0,0 +1,44 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Assets.Scripts.Events;
+using UnityEngine;
+
+namespace Assets.Scripts.Moves
+{
+    public abstract class MoveTrinket : TrinketBase
+    {
+        private readonly Events.EventHandler handler;
```

Comment:
I feel like this should be 
using Events;
EventHandler handler;

Reply:
Does github not allow you to make tasks on pull requests? I'm seriously considering moving this project to bitbucket